modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -10083,6 +10083,7 @@ language.mww=Hmong Daw
 language.nb=Norwegian
 language.nl=Dutch
 language.nl_BE=Dutch (Belgium)
+language.no_NO=Norwegian (Norway)
```

Comment:
Question: Why no only `language.no=...` ? There are no other language variation for this, right?